lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +159,32 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: test-telecom-id
+    path: /telecom/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the value of the `ContactPoint.id` of the telecom at the given <code>{index}</code> in the list of related person telecoms.
+    example: |
+      {
+        "op": "test",
+        "path": "/telecom/0/id",
+        "value": "CI-PH-2807950-0"
+      }
+
+  - name: remove-telecom
+    path: /telecom/{index}
+    operation: remove
+    type: N/A
+    url: https://hl7.org/fhir/R4/patient-definitions.html#related person.telecom
+    description: Remove the telecom at the given <code>{index}</code> in the list of related person telecoms.
+    example: |
+      {
+        "op": "remove"
+        "path": "/telecom/1"
```

Comment:
I think, since the other examples here are pointing to the 0 index, we can use the same index here, to keep things in sync.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/150edb0c78f14b75bf4da4f1412e24b65b90c50a